files/fr/glossary/continuous_media/index.html

```diff
@@ -0,0 +1,9 @@
+---
+title: Médias Continus
+slug: Glossary/Continuous_Media
+---
```

Comment:
I think we need a `translation_of: 'Glossary/Continuous_Media'` header here :)

Reply:
Thanks @audrasjb , do we have to always put this header within ``` ' ' ```? To correct it in my [previous PR](https://github.com/mdn/translated-content/pull/2056/files)

Reply:
Hello :) quotes are not mandatory when the string does not contain such string delimiters. (They don't hurt though)